Objects/Sensor.py

```diff
@@ -2,6 +2,27 @@
 import cv2
 from mss import mss
 from abc import ABC, abstractmethod
+import time
+
+
+import magichue
+
+
+TOKEN = 'eyJhbGciOiJSUzI1NiJ9.eyJhcHBLZXkiOiJaRzAwMSIsImNkcElkIjoiWkcwMDEiLCJjbGllbnRJZCI6Ik1ON1dBMVZZSjdBUFNHTTRNWDg3SENaNkQyMU5EWFAyIiwiZXhwaXJlRGF0ZSI6MTY5Mzc5MDkxMTY3OSwiZ3Vlc3QiOmZhbHNlLCJsb2dpbkRhdGUiOjE2NzgyMzg5MTE2NzksInJlZnJlc2hEYXRlIjoxNjg4NjA2OTExNjc5LCJzZXJ2ZXJDb2RlIjoiVVMiLCJ1aWQiOjE2MzMyMzU2ODAzOTQzMzgzMDYsInVzZXJJZCI6InNhbnRpYWdvajIyMkBnbWFpbC5jb20ifQ.n0GkAldq_xBzOgfnUskmBKWnAlSoujuzJs54WVP8KDpQ6AsJIBnvoqnxohbf-UBTrlwqlNiINT92irboO9e-fKc53dRqz-JQW3_C2FMls_lF5V10qE05fg91wfTKfvCJGdYTyqAKgV6SjCYucpb4O5DflbAlqhrAe49BE2HEw7nbGRnJsx9YMoO0ZZ6RpulF6CLqthz7qBl-0DgtQfTHecD3RJ7COvBl_qigFOvI8EHt83Mzjb77LT7dAlNW27oqC0115HFsW4duUujK4PxJPQhg9jKPY40DzTaiqYakHCZ7CkrJxGlGPScDGSk6Do5A_9CvcaYW7jRZT5LvZD2Caw'
```

Comment:
This is problematic. You usually don't want to hardcode tokens, nor share them with everyone on github. You will need to create another token, and for future reference, this is usually done with environment variables

Reply:
Use `.env` files for this, which are not included (use `.gitignore` for that), and include a `.env.example` that shows how to assign those variables